{
    "affected": [
        {
            "package": {
                "ecosystem": "Debian:8",
                "name": "python3.4",
                "purl": "pkg:deb/debian/python3.4?distro=jessie&repository_url=http%3A%2F%2Fdeb.freexian.com%2Fextended-lts"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "0"
                        },
                        {
                            "fixed": "3.4.2-1+deb8u11"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        }
    ],
    "database_specific": {
        "type": "security update"
    },
    "id": "ELA-510-1",
    "modified": "2021-11-05T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/ELA-510-1"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2021-3426"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2021-3733"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2021-3737"
        }
    ],
    "related": [
        "CVE-2021-3426",
        "CVE-2021-3733",
        "CVE-2021-3737"
    ]
}